test/powershell/Modules/Microsoft.PowerShell.Management/TimeZone.Tests.ps1

```diff
@@ -119,12 +122,15 @@ Describe "Get-Timezone test cases" -Tags "CI" {
 }
 
 Describe "Set-Timezone test case: call by single Id" -Tags @('CI', 'RequireAdminOnWindows') {
-    $originalTimeZoneId
     BeforeAll {
-        $originalTimeZoneId = (Get-TimeZone).Id
+        if ($IsWindows) {
```

Comment:
:confused: $IsNotSkipped is not used anywhere in the file. I do not see  inconsistent here.